Common subdirectories: Box2D-2.3b0/Box2D and Box2D-2.3b0.new/Box2D
Common subdirectories: Box2D-2.3b0/examples and Box2D-2.3b0.new/examples
Common subdirectories: Box2D-2.3b0/library and Box2D-2.3b0.new/library
diff -u Box2D-2.3b0/setup.py Box2D-2.3b0.new/setup.py
--- Box2D-2.3b0/setup.py	2013-02-02 18:09:34.000000000 +0100
+++ Box2D-2.3b0.new/setup.py	2014-10-25 13:32:07.136922343 +0200
@@ -176,7 +176,6 @@
     package_dir      = {'Box2D': library_path, 
                         'Box2D.b2': os.path.join(library_path, 'b2'),
                         'Box2D.tests' : 'tests'},
-    test_suite       = 'tests',
     options          = { 'build_ext': { 'swig_opts' : swig_arguments },
                          'egg_info' : { 'egg_base' : library_base },
                         },
